lib/rucio/common/config.py

```diff
@@ -171,25 +171,27 @@ def config_get(
     First it looks at the configuration file and, if it is not found, check in the config table only if it is called
     from a server/daemon (and if check_config_table is set).
 
+    Note: When a non-None default is provided, the raise_exception parameter is redundant and should be omitted.
+          Similarly, if raise_exception is True, it can be omitted since that is the default behavior.
+
     :param section: the named section.
     :param option: the named option.
-    :param raise_exception: Boolean to raise or not NoOptionError, NoSectionError or RuntimeError.
+    :param raise_exception: Boolean to raise or not NoOptionError, NoSectionError or RuntimeError;
+                            only used when no default is provided.
     :param default: the default value if not found.
     :param clean_cached: Deletes the cached config singleton instance if no config value is found
-    :param check_config_table: if not set, avoid looking at config table even if it is called from server/daemon
-    :param session: The database session in use. Only used if not found in config file and if it is called from
-                    server/daemon
-    :param use_cache: Boolean if the cache should be used. Only used if not found in config file and if it is called
-                      from server/daemon
-    :param expiration_time: Time after that the cached value gets ignored. Only used if not found in config file and if
-                            it is called from server/daemon
+    :param check_config_table: if not set, avoid looking at config table even if it is called from server/daemon;
+                               only used if not found in the config file.
```

Comment:
The original docstring doesn't mention 'only used if not found in the config file.' for `check_config_table`, so this should be removed